modules/server/src/network/queries/index.ts

```diff
@@ -21,6 +22,13 @@ export type GQLTypeQueryResponse = {
 	};
 };
 
+// Data query response
+export type GQLResponse = {
```

Comment:
I think this name is too generic, there are lots of GQL apis and they don't all have a response in this form.